src/sentry/integrations/vsts/client.py

```diff
@@ -17,15 +17,15 @@
 
 
 class VstsApiPath(object):
-    commits = u'{instance}DefaultCollection/_apis/git/repositories/{repo_id}/commits'
-    commits_batch = u'{instance}DefaultCollection/_apis/git/repositories/{repo_id}/commitsBatch'
-    commits_changes = u'{instance}DefaultCollection/_apis/git/repositories/{repo_id}/commits/{commit_id}/changes'
-    projects = u'{instance}DefaultCollection/_apis/projects'
-    repository = u'{instance}DefaultCollection/{project}_apis/git/repositories/{repo_id}'
+    commits = u'{instance}_apis/git/repositories/{repo_id}/commits'
+    commits_batch = u'{instance}_apis/git/repositories/{repo_id}/commitsBatch'
+    commits_changes = u'{instance}_apis/git/repositories/{repo_id}/commits/{commit_id}/changes'
+    projects = u'{instance}_apis/projects'
+    repository = u'{instance}{project}_apis/git/repositories/{repo_id}'
     repositories = u'{instance}{project}_apis/git/repositories'
     subscription = u'{instance}_apis/hooks/subscriptions/{subscription_id}'
     subscriptions = u'{instance}_apis/hooks/subscriptions'
-    work_items = u'{instance}DefaultCollection/_apis/wit/workitems/{id}'
+    work_items = u'{instance}_apis/wit/workitems/{id}'
```

Comment:
Does `instance` end in a `/`?

Reply:
I believe the base URL we get back from Azure includes the `/`.